vmware_exporter/helpers.py

```diff
@@ -1,8 +1,20 @@
 import os
 
+import requests
 from pyVmomi import vmodl
 
 
+def get_unverified_session():
+    """
+    create session to connect to vmware API
+    from Vmware Samples
+    """
+    session = requests.session()
+    session.verify = False
```

Comment:
Can you make this a defaulted variable?

Reply:
I am pushing a version where I am replacing vmware SDK for direct requests to vcenter REST API so as to fetch objects tags. So this bit of code won't exist.